SWADroid/src/main/res/drawable/togglebuttonf.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
You should use a more descriptive name for the file. Since the buttons only differ in color, maybe you could include the color in the file name.

Would it be possible to refactor these buttons into a single drawable that could be colored programmatically?